client/my-sites/sidebar/sidebar.jsx

```diff
@@ -411,28 +364,24 @@ export class MySitesSidebar extends Component {
 	}
 
 	users() {
-		var site = this.getSelectedSite(),
-			usersLink = '/people/team' + this.siteSuffix(),
-			addPeopleLink = '/people/new' + this.siteSuffix();
-
-		if ( ! site.capabilities ) {
-			return null;
-		}
+		const { site, canUserListUsers } = this.props;
+		let usersLink = '/people/team' + this.props.siteSuffix;
+		let addPeopleLink = '/people/new' + this.props.siteSuffix;
 
-		if ( site.capabilities && ! site.capabilities.list_users ) {
+		if ( site && ! canUserListUsers ) {
 			return null;
 		}
 
-		if ( ! this.isSingle() ) {
+		if ( ! this.props.siteId ) {
```

Comment:
Is this equivalent to a single site check? We could maybe pass through `isSingleSite` in props